web/src/main/java/org/cbioportal/web/parameter/ClinicalDataBinFilter.java

```diff
@@ -51,12 +52,12 @@ public void setEnd(Double end) {
     private boolean rangeIsCoveringCustomBins() {
         if (this.customBins != null && (start != null || end != null)) {
             boolean valid = true;
-            for (Double bin : this.customBins) {
-                if(start != null && start > bin) {
+            for (BigDecimal bin : this.customBins) {
+                if(start != null && start.compareTo(bin) == 1) {
```

Comment:
maybe we should use `> 0` instead of `== 1`: https://stackoverflow.com/a/34677687

Reply:
@onursumer I think it's fine. The compareTo only produces -1, 0, 1.